components/forms/Description/Description.tsx

```diff
@@ -11,11 +11,18 @@ export const Description = (props: DescriptionProps): React.ReactElement => {
   const { children, className, id } = props;
 
   const classes = classnames("gc-description", className);
-
   return (
-    <p data-testid="description" className={classes} id={`desc-${id}`}>
-      {children}
-    </p>
+    <div id={`desc-${id}`}>
+      {children
+        ? children.split("<br>").map((child, index) => {
+            return (
+              <p data-testid="description" className={classes} key={index}>
```

Comment:
Strange to replace `<br>` with `<p></p>`, as they are semantically differently tags. Is it possible to split on `<br>` within the string, as you are currently doing, and replace it with an actual `<br>` tag?

Reply:
Good idea.  Implemented